pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -531,6 +531,12 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 					}
 				}
 
+				// Don't backup PVCs with skip resource annotation set
+				if resourcecollector.SkipResource(pvc.Annotations) {
```

Comment:
we should add this in resource collector, that way other components such as px-central-middleware can skip resources while listing in UI

Reply:
resourcecollector.SkipResource is also checking for the same annotation  "stork.libopenstorage.org/skip-resource" and the setting  of this annotation happens in reconciler code of kdmp controller.

Reply:
@ram-infrac You will still need the check here since we are getting the list of PVCs in this case from k8s api-server.